sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -434,6 +434,30 @@ abstract class StreamExecution(
     }
   }
 
+  /**
+   * Interrupts the query execution thread and awaits its termination until the
```

Comment:
> until the `spark.sql.streaming.stopTimeout`

That seems to be ended oddly. I'd rather reword as "until it exceeds the timeout.", and maybe optionally describe "Timeout can be set on ``spark.sql.streaming.stopTimeout`.". Given this method is not exposed by end users, I'd assume we might not need to explicitly describe configuration in scaladoc.